src/object/spotlight.cpp

```diff
@@ -42,6 +45,8 @@ Spotlight::Spotlight(const ReaderMapping& lisp) :
   lisp.get("speed", speed, 50.0f);
   lisp.get("counter-clockwise", counter_clockwise, false);
 
+  emitting=true;
```

Comment:
Instead of writing ```emitting = true```, please initialize this in the initialization list, behind ```speed(50.0f),``` like in the following example:
```
emitting(true),
```